src/project/qml/MuseScore/Project/internal/Export/ExportOptionsView.qml

```diff
@@ -183,4 +187,6 @@ Column {
             exportModel.shouldDestinationFolderBeOpenedOnExport = !checked
         }
     }
+
+    Component.onCompleted: root.refreshPageLoader()
```

Comment:
Our convention is to write `Component.onCompleted` near the top of the file; in this case, I'd put it between `spacing: 12` (line 37) and `NavigationPanel {`.

Also, we prefer to use braces, to make clear that this is a function rather than a property value, like this:
```suggestion
    Component.onCompleted: {
        root.refreshPageLoader()
    }
```